test/format/trace.c

```diff
@@ -43,17 +43,16 @@ void
 trace_config(const char *config)
 {
     char *copy, *p;
-    bool all;
 
     copy = dstrdup(config);
-    for (all = false;;) {
-        TRACE_CONFIG_CMD("all", all);
-        TRACE_CONFIG_CMD("bulk", g.trace_bulk);
-        TRACE_CONFIG_CMD("cursor", g.trace_cursor);
-        TRACE_CONFIG_CMD("mirror_fail", g.trace_mirror_fail);
-        TRACE_CONFIG_CMD("read", g.trace_read);
-        TRACE_CONFIG_CMD("timestamp", g.trace_timestamp);
-        TRACE_CONFIG_CMD("txn", g.trace_txn);
+    for (;;) {
```

Comment:
Maybe, to be on the safe side we should init the trace flags before the for loop? Something like `g.trace_flags = 0;`, what do you think?